cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@joystream/cli",
   "description": "Command Line Interface for Joystream community and governance activities",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
I'm wondering if we should bump versions of all packages that still start with `0.` to `1.` now that we're on mainnet,

Reply:
I have no objection.

Reply:
1.0.0 - mainnet
1.1.0 - apps
1.20 - ephesus

@Lezek123

Reply:
If you mean `1.2.0` for Ephesus then sounds good.

Reply:
yep, typo there :)